install_config/router/default_haproxy_router.adoc

```diff
@@ -1566,6 +1539,31 @@ haproxy_server_bytes_in_total{namespace="default",pod="hello-rc-vkjqx",route="he
 ...
 ----
 
+* To get an HTML file with metrics, delete the following xref:../../architecture/networking/routes.adoc#env-variables[environment variables]
+from the router deployment configuration file:
++
+----
+$ oc edit service router
+
+- name: ROUTER_LISTEN_ADDR
+  value: 0.0.0.0:1936
+- name: ROUTER_METRICS_TYPE
+  value: haproxy
+----
++
+Launch the stats window using the following URL in a browser, where 1936 is the `STATS_PORT` value:
```

Comment:
I think the organization of this section could be improved. I think this section is actually three substeps and that you have them presented in 1-3-2 order.
s/Launch the stats window using the following URL in a browser, where 1936 is the `STATS_PORT` value/Open the stats window at the following URL:
The note about 1936 might be better as an annotation on line 1557.